components/onboarding_tasks/onboarding_tasks_manager.tsx

```diff
@@ -39,65 +41,100 @@ import {
-        />
-    ),
-    [OnboardingTasksName.DOWNLOAD_APP]: (
-        <FormattedMessage
-            id='onboardingTask.checklist.task_download_mm_apps'
-            defaultMessage='📱 Download the Desktop and Mobile Apps'
-        />
-    ),
-    [OnboardingTasksName.VISIT_SYSTEM_CONSOLE]: (
-        <FormattedMessage
-            id='onboardingTask.checklist.task_visit_system_console'
-            defaultMessage='🥰️ Visit the System Console to configure your workspace'
-        />
-    ),
-    [OnboardingTasksName.START_TRIAL]: (
-        <FormattedMessage
-            id='onboardingTask.checklist.task_start_enterprise_trial'
-            defaultMessage='🏢 Learn more about Enterprise-level high-security features'
-        />
-    ),
+const TaskListImage = styled.img`
+    width: 24px;
+    height: 24px;
+`;
+
+// eslint-disable-next-line react-hooks/rules-of-hooks
+
+interface OnBoardingTaskListTaskProps {
+    id: string;
+    svg: string;
+    message: string;
+}
+
+const getTaskDetails = {
+    [OnboardingTasksName.CHANNELS_TOUR]: {
+        id: 'onboardingTask.checklist.task_learn_more_about_messaging',
+        svg: channels,
+        message: 'Take a tour of Channels.',
+    },
+    [OnboardingTasksName.BOARDS_TOUR]: {
+        id: 'onboardingTask.checklist.plan_sprint_with_kanban_style_boards',
+        svg: bullseyeImg,
+        message: 'Manage tasks with your first board.',
+    },
+    [OnboardingTasksName.PLAYBOOKS_TOUR]: {
+        id: 'onboardingTask.checklist.task_resolve_incidents_faster_with_playbooks',
+        svg: clipboard,
+        message: 'Explore workflows with your first playbook.',
+    },
+    [OnboardingTasksName.INVITE_PEOPLE]: {
+        id: 'onboardingTask.checklist.task_invite_team_members',
+        svg: handshake,
+        message: 'Invite team members to the workspace.',
+    },
+    [OnboardingTasksName.COMPLETE_YOUR_PROFILE]: {
+        id: 'onboardingTask.checklist.task_complete_your_profile',
+        svg: sunglasses,
+        message: 'Complete your profile.',
+    },
+
+    [OnboardingTasksName.EXPLORE_OTHER_TOOLS]: {
+        id: 'onboardingTask.checklist.explore_other_tools_in_platform',
+        svg: wrench,
+        message: 'Explore other tools in the platform.',
+    },
+
+    [OnboardingTasksName.DOWNLOAD_APP]: {
+        id: 'onboardingTask.checklist.task_download_mm_apps',
+        svg: phone,
+        message: 'Download the Desktop and Mobile Apps.',
+    },
+
+    [OnboardingTasksName.VISIT_SYSTEM_CONSOLE]: {
+        id: 'onboardingTask.checklist.task_visit_system_console',
+        svg: gears,
+        message: 'Visit the System Console to configure your workspace.',
+    },
+    [OnboardingTasksName.START_TRIAL]: {
+        id: 'onboardingTask.checklist.task_start_enterprise_trial',
+        svg: security,
+        message: 'Learn more about Enterprise-level high-security features.',
+    },
+};
+
+const useOnBoardingTaskListTask = ({id, svg, message}: OnBoardingTaskListTaskProps) => {
```

Comment:
this is not needed as a hook, the name 'use...' is normally intended to be used as a hook, and since this seems to be just a normal function that returns some JSX depending on the parameters, either change the function signature name or make it a React component and use it as a React component.